### DISCUSSION TITLE
AWS Kinesis Video Streams

### DISCUSSION THREAD

**Discussion by currydem** [aiortc/aiortc#858] [Q&A]
I just installed the aiortc library on my raspberry pi 4. I am using pi camera and trying to stream to my AWS Kinesis Video Streams. I have seen tutorials on how to use the standard WebRTC with AWS Kinesis Video Streams. Is this library able to do this as well? And if so, how do I go about accomplishing this?

**Comment by CrabbyPete** [comment]
Did you ever get this to work. I am trying it on my RPIZero2
